lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1233,8 +1228,12 @@ def get_service_name_from_init_system(self, service):
             (str) service name
```

Comment:
Initially I did not get what this function is doing. IMHO a better name for this function is:
`map_service_to_init_system_service_name`

Reply:
Changed.